public/manageclaims.php

```diff
@@ -0,0 +1,199 @@
+<?php
+require_once __DIR__ . '/../vendor/autoload.php';
+require_once __DIR__ . '/../lib/bootstrap.php';
+
+use RA\ArticleType;
+use RA\Permissions;
+
+if (!authenticateFromCookie($user, $permissions, $userDetails, Permissions::Admin)) {
+    header("Location: " . getenv('APP_URL'));
+    exit;
+}
+
+$gameID = requestInputSanitized('g', null, 'integer');
+
+if (empty($gameID)) {
+    header("Location: " . getenv('APP_URL'));
+}
+
+$claimData = getFilteredClaimData($gameID);
+getGameMetadata($gameID, $user, $achievementData, $gameData);
```

Comment:
if you don't need `$achievementData`, you can call a lighter function to get `$gameData`:
```
$gameData = getGameData($gameID);
```